### DISCUSSION TITLE
Keep one month worth of data - gc_num_epochs_to_keep

### DISCUSSION THREAD

**Discussion by andreclaro** [near/nearcore#11692] [Q&A]
We have a requirement to keep one month worth of historical data in our Near RPC nodes (testnet and mainnet).

I think we can archive that by setting `gc_num_epochs_to_keep` to 60, since 1 epoch is around 12 hours.

Can you confirm this is the way to do it? Thanks!

**Comment by TatyOko28** [comment]
Yes, setting gc_num_epochs_to_keep to 60 should retain approximately one month of historical data, since each epoch lasts around 12 hours (60 epochs × 12 hours = 30 days). However, ensure your storage capacity can handle the additional data, as keeping more epochs increases disk usage. You may also want to test on testnet before applying it to mainnet to verify performance and consistency. Let us know if you need further clarification!
